spec/truemail/validate/smtp/request_spec.rb

```diff
@@ -42,16 +42,15 @@
 
     context 'when port opened' do
```

Comment:
insert to 9 line: `port_open_status: port_open_status,`
insert to 22 line: `let(:port_open_status)       { proc { true } }`
change '#check_port' block in 42 line:
```ruby
describe '#check_port' do
  let(:response_instance_target_method) { request_instance.check_port }

  context 'when port opened' do
    specify do
      allow(Socket).to receive(:tcp)
        .with(
          mail_server,
          Truemail::Validate::Smtp::Request::SMTP_PORT,
          connect_timeout: connection_timeout
        ) { |&block| expect(block).to eq(port_open_status) }
        .and_return(true)
      expect { request_instance.check_port }
        .to change(response_instance, :port_opened).from(nil).to(true)
    end
  end

  context 'when port closed' do
    let(:error_stubs) do
      allow(Socket).to receive(:tcp)
        .with(
          mail_server,
          Truemail::Validate::Smtp::Request::SMTP_PORT,
          connect_timeout: connection_timeout
        )
        .and_raise(Errno::ETIMEDOUT)
    end

    specify do
      error_stubs
      expect { response_instance_target_method }.to change(response_instance, :port_opened).from(nil).to(false)
    end

    include_examples 'request retry behavior'
  end
end
```